pages/index.js

```diff
@@ -1,12 +1,18 @@
 import React from 'react'
 import Link from 'next/link'
 
-export default () => (
-  <main>
-    <h1>Hello from Preact</h1>
+const Home = () => {
```

Comment:
Due to `export default () => ` having no name this is needed to make `prefresh` recognise this as a component as said in our [best practices section](https://github.com/JoviDeCroock/prefresh#best-practices)

Reply:
Seems fine. I'd love to have this be able to infer components from the existence of JSX within them, but this is a very reasonable compromise.